plugins/unbound-pvc.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: unbound-pvc       # plugin name must match your manifest file name (e.g. foo.yaml)
+spec:
+  version: "v0.0.1"       # required, must be in semver format, prefixed with "v"
+  platforms:
+  - selector:             # a regular Kubernetes selector
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    # url for downloading the package archive:
+    uri: https://github.com/ishantanu/kubectl-unbound-pvc/archive/v0.0.1.zip
+    # sha256sum of the above archive file:
+    sha256: "36231dcd6fbacd90c4929d75bff119d8ef1a2dbcec2b0baca639edfd489ecf4a"
+    # copy the used files out of the zip archive, defaults to `[{from: "*", to: "."}]`
+    files:
+    - from: "./kubectl-unbound-pvc-0.0.1/kubectl-unbound_pvc" # path to the files extracted from archive
```

```suggestion
    - from: "./kubectl-unbound-pvc-*/kubectl-unbound_pvc" # path to the files extracted from archive
```

Like that, version bumps can be auto-approved in the future.

---

Fixed in b4166b4